src/test/java/hw6/ex2/RunTests.java

```diff
@@ -0,0 +1,8 @@
+package hw6.ex2;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.testng.AbstractTestNGCucumberTests;
+
+@CucumberOptions(glue = "hw6")
```

Comment:
Code duplication. I could not see any different from the `hw6.ex1.RunTests` class

Reply:
Solved: I removed this unnecessary file